docs/rtc/index.md

```diff
@@ -0,0 +1,47 @@
+# Real Time Collaboration
+
+JupyterLite enables basic support for Real Time Collaboration over WebRTC.
+
+## What is Real Time Collaboration
+
+Real Time Collaboration (RTC for short) makes it possible for users to collaborate on
+the same document at the same time.
+
+Here is an example screencast of what it looks like in JupyterLab:
+
+![rtc-demo](https://user-images.githubusercontent.com/591645/117701750-e6940280-b1c7-11eb-92e6-2ce0331febeb.gif)
+
+Since JupyterLite reuses most of the upstream JupyterLab components and plugins, it also
+gets support for RTC almost automatically.
+
+The main differences compared to JupyterLab are:
+
+- the user content is decentralized: users store notebooks and files in their own
+  browser local storage
+- the communication with peers and the initial syncing mechanism happens over WebRTC,
+  instead of the WebSocket Jupyter Server handler in JupyterLab
+
+## Enabling RTC in JupyterLite
+
+To enable RTC in JupyterLite, you need to set the `collaborative` flag in the
+`jupyter-config-data`. This follows the same behavior as in JupyterLab 3.1 where real
+time collaboration is an opt-in feature.
+
+In addition to the `collaborative` flag, end users must specify the `room` query
+parameter in the URL. An example of such URL is as follows:
+
+[https://jupyterlite.readthedocs.io/en/latest/\_static/lab/index.html?room=my-custom-room](https://jupyterlite.readthedocs.io/en/latest/_static/lab/index.html?room=my-custom-room)
```

Comment:
i wouldn't even make this a clickable link... and frankly, we shouldn't even suggest a room name, as we basically _don't_ want people going to The One Room because it will get filled up with spam.

A nice approach might be to offer an RTC generator with an in-page widget... though `StringConcatenate` isn't a thing, and `wxyz` doesn't work with `requirejs`

Reply:
Good point. We can still show to specify the `room` parameter, but it could be something else or just a random string.

> A nice approach might be to offer an RTC link generator with an in-page widget

Yes that would make it much more convenient to share a link.

Reply:
An in-app one would be clutch, too, but could be a federated extension (let's not make webpack any worse until we make it better).

In this _specific_ instance, I mean a link generator _on the docs page_, akin to the binder one, where we can hoist all of the bits and bobs.